app/api/api_v1/routers/admin.py

```diff
@@ -317,6 +323,69 @@ def _start_ingest(
         )
 
 
+@r.post(
+    "/bulk-ingest/validate/cclw/law-policy",
+    response_model=ValidationResult,
+    status_code=status.HTTP_200_OK,
+)
+def validate_law_policy(
+    request: Request,
+    law_policy_csv: UploadFile,
+    db=Depends(get_db),
+    current_user=Depends(get_current_active_superuser),
+):
+    """
+    Validates the provided CSV into the document / family / collection schema.
+
+    :param [Request] request: Incoming request (UNUSED).
+    :param [UploadFile] law_policy_csv: CSV file to ingest.
+    :param [Session] db: Database connection.
+        Defaults to Depends(get_db).
+    :param [JWTUser] current_user: Current user.
+        Defaults to Depends(get_current_active_superuser).
+    :return [str]: A path to an s3 object containing document updates to be processed
+        by the ingest pipeline.
+    :raises HTTPException: The following HTTPExceptions are raised on errors:
+        400 If the provided CSV file fails schema validation
+        422 On failed validation on the input CSV (results included)
+        500 On an unexpected error
+    """
+
+    _LOGGER.info(
+        f"Superuser '{current_user.email}' triggered Bulk Document Validation for "
+        "CCLW Law & Policy data"
+    )
+
+    try:
+        context = db_init(db)
+    except Exception as e:
+        _LOGGER.exception(
+            "Failed to create ingest context", extra={"props": {"errors": str(e)}}
+        )
+        raise HTTPException(status_code=status.HTTP_500_INTERNAL_SERVER_ERROR) from e
+
+    all_results = []
+
+    try:
+        _, message = validate_law_policy_csv(law_policy_csv, db, context, all_results)
```

Comment:
Just to check, the _ is not used and is just here to deconstruct the response?

Reply:
Yup